src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -385,6 +466,11 @@ def get_stats_data_for_trending_events(results):
                 trends_requests,
             ) = get_trends_data(stats_data, request)
 
+            if features.has(
+                "organizations:performance-trends-issues", organization, actor=request.user
+            ):
+                send_occurrence_to_plaform(trending_events)
```

Comment:
I'd wrap this in a `try ... except ...` in the unlikely case it errors so it doesn't break the trends page.